tools/cg/about.toml

```diff
@@ -0,0 +1,9 @@
+accepted = [
```

Comment:
Freyja also includes `ISC`, `EPL-1.0`, and `EPL-2.0` in this list. it may also be beneficial to sort alphabetically